manifests/charts/base/crds/crd-operator.yaml

```diff
@@ -11,6 +11,10 @@ spec:
     description: Istio control plane revision
     name: Revision
     type: string
+  - JSONPath: .status.status
+    description: IOP current state
```

Comment:
nit- I don't think "IOP" is a user facing term, lets us something like "current status of the control plane"